features/importing/work_import.feature

```diff
@@ -146,6 +146,63 @@ Feature: Import Works
     Then I should see "Chapters:2/2"
       And I should see "IP Address: 127.0.0.1"
 
+  Scenario: Imported works can be set to restricted
+    When I start importing "http://import-site-with-tags" with a mock website
+      And I check "restricted"
```

Comment:
Let's use the user-facing text when choosing our settings. That will help us know we don't have any missing translations.